tox.ini

```diff
@@ -34,7 +34,7 @@ basepython = python3
 usedevelop = false
 deps = flake8
 changedir = {toxinidir}
-commands = flake8 .
+commands = flake8 django tests
```

Comment:
Looks like you forgot to revert this change before creating this PR

Reply:
Whoops, thanks.